src/index.ts

```diff
@@ -552,7 +553,17 @@ export class Loader {
       /* eslint-enable */
     }
 
-    this.importLibrary("core").then(
+    // While most libraries populate the global namespace when loaded via bootstrap params,
+    // this is not the case for "marker" when used with the inline bootstrap loader
+    // (and maybe others in the future). So ensure there is an importLibrary for each:
+    const libraryPromises = this.libraries.map((library) =>
+      this.importLibrary(library)
```

Comment:
importLibrary is async... could there be a race condition if you don't wait for the Promise to be resolved?